### ISSUE TITLE
Metatag Version

### ISSUE THREAD

**Issue by RealLukeMartin** [poetic/rune#97]
The current dev release of metatag, that we are running on, is causing issues with migrations and clones on Aegir. Aegir shows incompatibility with versions inbetween platforms although they have the same 1.x-dev-dev versions. 

Is there a reason we want to run off 1.x-dev-dev release or can we go ahead and run off stable release of metatag?

**Comment by RealLukeMartin** [comment]
The pertinence:

```
  metatag:
    download: { branch: 1.x-dev, revision: 3a7295d }
    patch:
      - https://www.drupal.org/files/issues/metatag-substitute-panels-context-variables-2408211-8.patch
      - https://www.drupal.org/files/issues/metatag-feeds-module-api-change-2475147-4.patch

```

**Comment by RealLukeMartin** [comment]
Found out both the patches were merged into 1.5. Going to upgrade metatag to at least that for now and test 1.7.
